### ISSUE TITLE
Publishing to directory /bin/staticsites/ss-oryx/app... takes > 10 minutes

### ISSUE THREAD

**Issue by darena-patrick** [Azure/static-web-apps#1000]
**Describe the bug**

When publishing a Blazor WebAssembly (WASM) application to Static Web App, the step `Publishing to directory /bin/staticsites/ss-oryx/app...` takes 800s+ ... is there any way to speed this up? It is burning up valuable GitHub runner minutes.

**To Reproduce**
1. Create Static Web App via Azure Portal
2. Connect it to GitHub Blazor project
3. Run GitHub Action

Using the exact generated `.yaml` file with no modifications other than what was automatically done through the Azure Portal configuration.

**Expected behavior**
This should take only a few seconds or couple of minutes maximum. 

**Screenshots**
![image](https://user-images.githubusercontent.com/77751400/203665792-d4cba000-8381-4320-afb9-c93783b1f077.png)

**Comment by darena-patrick** [comment]
For reference, running a second time is also very long ~12m 
![image](https://user-images.githubusercontent.com/77751400/203666760-ea2e990f-a285-4b58-8653-08e620917dcb.png)

**Comment by darena-patrick** [comment]
For reference, I did the exact same thing with a sample project and it only took 85s. What is happening at this step that could be taking a long time?

- Is it the file size of the project?
- Could it be the quantity of warnings generated by the build process?
- What is actually happening at this step? I turned on debug logging, but it doesn't log any extra logs for this step to be able to see what is happening. Is there any other way to dig deeper to figure out what I need to modify in my project to make it as fast as the sample project?
